### PR TITLE
Feature/topScorers

### PR DESCRIPTION
**Pull request by kiasha10**
[Feature/Merge] - TopScorers Entry page 

[Merge] Feature/topScorers → dev
[Feature] - TopScorers

 ## Description
Added UI Changes 
Added a segue button - will do functionality in next PR

## Checklist
Please review and check the following before submitting this pull request:
- [ ] The code follows the project's coding standards.
- [ ] 100% Unit test coverage for View Models

### CODE REVIEW

**Review by AShozi — DISMISSED** [review]
looks solid, nice job

File: BlitzBoard/Features/TopScorersScreenViewController.swift [hunk 1]
```diff
@@ -1,13 +1,83 @@
...
+    }
+}
+extension TopScorersScreenViewController: ViewModelDelegate {
+    
+    func reloadView() {
+            self.tableView.reloadData()
+    }
+    
+    func show(error: String) {
+
```
Comment (ksookdhew) [inline BlitzBoard/Features/TopScorersScreenViewController.swift:81]: Is this meant to be empty 😅

File: BlitzBoard/Features/TopScorersScreenViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+    
+    func fetchTopScorers() {
+        repository.fetchTopScorers { [weak self] result in
+            switch result {
+            case .success(let fetchedPlayers):
+                self?.topScorers = fetchedPlayers
+                self?.delegate?.reloadView()
+            case .failure(let error):
+                print("Error: \(error)")
```
Comment (ksookdhew) [inline BlitzBoard/Features/TopScorersScreenViewModel.swift:42]: Consider to calling your delegate show error function here

**Review by AShozi — APPROVED** [review]

**Review by giacatano — APPROVED** [review]

**Review by ksookdhew — APPROVED** [review]
Nice 👏

**Pull request merged**